src/Spec2-Core/String.extension.st

```diff
@@ -1,9 +1,32 @@
 Extension { #name : #String }
 
+{ #category : #'*Spec2-Core' }
+String >> accentuateCharacter: aCharacter [
```

Comment:
if you are answering a copy (or a new instance), I prefer something like `withAccentuatedCharacter:` as name.

Reply:
renamed